3MI0800303/practicum-1.cpp

```diff
@@ -0,0 +1,148 @@
+#include <iostream>
+#pragma warning(disable : 4996)
+
+struct OrderedTwo {
+    int firstNumber;
+    int secondNumber;
+    bool isBigger(OrderedTwo& ordered) const;
+};
+
+inline bool OrderedTwo::isBigger(OrderedTwo& cmp) const {
+    if (firstNumber == cmp.firstNumber) return secondNumber > cmp.secondNumber;
+    if (firstNumber < cmp.firstNumber) return false;
+    return true;
+}
+
+int getBiggest(OrderedTwo ords[100]) {
+    for (size_t i = 0; i < 100; i++) {
+        for (size_t j = i + 1; j < 100; j++) {
+            if (ords[i].isBigger(ords[j])) std::swap(ords[i], ords[j]);
+        }
+    }
+
+    return 0;
+}
+
+enum class Course {
+    first,
+    second,
+    third,
+    fourth
+};
+
+struct Student {
+    char name[16];
+    int fn_number;
+    Course course;
+    double avg_grade;
+    void printStudents() const;
+};
+
+void Student::printStudents() const {
+    char courseChar[7];
```

Comment:
It would've been great to extract the conversion of an `enum element` to "string" in a separate function.